test/v1/slo/slo-with-annotations.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: openslo/v1
+kind: SLO
+metadata:
+  name: monthy-openslo-slo
+  displayName: Python
+spec:
+  #annotation:nobl9-kind:Direct
```

Comment:
I'm afraid this won't cover my use-case.
Annotation is currently stored as a comment in yaml which can't be generated using SLO structs.

Would it be possible to replace `#annotation:nobl9-kind:Direct` with
```yaml
metadata:
  annotations:
    nobl9-kind: Direct
```

---

Just a bit more about use-case I've.
I've plenty of services which are quite similar, so for example availability `SLOs` are almost the same for each of the apps I've. I'm not crafting any of openSLO files manually, but I'm generating those using go structs.  This makes it important for me to be able to generate relevant config using structs which can be mapped directly to `oslo` yaml files.

Reply:
Yes, it looks like better solution :D my bad! I forget about that option, I'll prepare something else in couple of days